nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ExecuteSQL.java

```diff
@@ -125,14 +125,14 @@ public ExecuteSQL() {
 
     @Override
     public void onTrigger(final ProcessContext context, final ProcessSession session) throws ProcessException {
-        FlowFile incoming = null;
+        FlowFile fileToProcess = null;
```

Comment:
Was confused at first the change to only have one flowfile but this is definitely an improvement to change to only process one flowfile. Prior it would take a flowfile and then it would magically disappear. The user would have no idea where it went or what happened.
Good catch Matt!